packages/client/wallets/aa/src/SmartWalletSDK.ts

```diff
@@ -17,6 +18,7 @@ export class SmartWalletSDK extends LoggerWrapper {
     private readonly eaoWalletService: EOAWalletService;
 
     private constructor(config: SmartWalletSDKInitParams) {
+        logger.setDDLoggerOnLocalhost(Boolean(config.logging));
```

Comment:
`Boolean(config.logging)` is kinda weird, more idiomatic would be:
```suggestion
        logger.setDDLoggerOnLocalhost(config.logging === true);
```

Reply:
Rather than decide "yes I want logging" or "no I don't want logging", much more extensible in my mind to be able to provide a logger.
Don't want logging? => provide `undefined`, omit
Want logging to console? => provide `ConsoleLogger` (maybe we need to provide)
Want logging to your production logs but not the console? => Provide a custom logger that hooks up to your service